src/Interactive/EditorFeatures/Core/Implementation/Interactive/Commands/ClearScreenCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Interactive;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.InteractiveWindow.Commands;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+{
+    [Export(typeof(IInteractiveWindowCommand))]
+    [InteractiveWindowRole(InteractiveWindowRoles.Any)]
+    internal sealed class ClearScreenCommand : InteractiveWindowCommand
+    {
+        public override Task<ExecutionResult> Execute(IInteractiveWindow window, string arguments)
+        {
+            window.Operations.ClearView();
+            return ExecutionResult.Succeeded;
+        }
+
+        public override string Description
+        {
+            get { return "Clears the contents of the REPL editor window, leaving history and execution context intact."; }
```

Comment:
It's hard-coded in the copy of this command that ships in Dev14 RTM.  Would ```//TODO``` suffice for now?

(I hope this whole class can go away once we have a unified way of filtering between us/Python)